app/models/user_target.rb

```diff
@@ -1,4 +1,12 @@
 class UserTarget < ApplicationRecord
   belongs_to :user
   belongs_to :topic
+
+  validates :title, :user, :topic, :latitude, :longitude, :area, presence: true
+
+  DEFAULT_AREAS = [50, 100, 150, 200]
+
+  def self.default_areas
```

Comment:
is this method necessary? you can do `UserTarget:: DEFAULT_AREAS` to get the default areas